skt/runner.py

```diff
@@ -53,11 +52,12 @@ def __call__(self, task, waiving, is_task_waived_func, prev_task):
         task_results = {
             'result': task.attrib.get('result'),
             'status': task.attrib.get('status'),
-            'waived': True if waiving and is_task_waived_func(task) else False,
+            'waived': True if is_task_waived_func(task) else False,
```

Comment:
Question: do we want to _not_ assume that the function `it_task_waived_func` will always return `True` or `False`? Because otherwise it seems cleaner to say:
```
'waived': is_task_waived_func(task)
```
I have checked the code and the only function currently used as `is_task_waived_func` is `is_task_waived` (defined in `misc.py`), and that function always returns `True` or `False`, so...

Reply:
Good point! I'll refresh the PR later (together with the rebase, if Jakub's PR will be merged while I return)

Reply:
Fixed.